lib/job/run.js

```diff
@@ -20,6 +20,7 @@ module.exports = function() {
     self.computeNextRunAt();
     await self.save();
 
+    let lockTimeout = null;
```

Comment:
I'd love to see a comment explaining why this variable exists, it's purpose. I can't quite understand from the code. Sorry.